tests/activityDetection.test.js

```diff
@@ -3,10 +3,10 @@ const childProcess = require('child_process');
 
 let activityDetection;
 
-describe('Tests for Activity Detection', function() {
+describe('Tests for Activity Detection', function () {
 
     var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
```

Comment:
I'll approve this for now. We'll need to find a soln. though. Please create a new PR once you find a fix.